<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>SettingsWindow</class>
 <widget class="QWidget" name="SettingsWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>880</width>
    <height>551</height>
   </rect>
  </property>
  <layout class="QHBoxLayout" name="horizontalLayout" stretch="1,3">
   <item>
    <layout class="QVBoxLayout" name="verticalLayout">
     <item>
      <widget class="QLabel" name="titleLabel">
       <property name="styleSheet">
        <string notr="true">font-size: 18px; font-weight: 600;</string>
       </property>
       <property name="text">
        <string>Settings</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="btnWallpaper">
       <property name="text">
        <string>Wallpaper</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QListWidget" name="listWidget"/>
     </item>
    </layout>
   </item>
   <item>
    <widget class="QStackedWidget" name="stackedWidgetSettings">
     <property name="enabled">
      <bool>true</bool>
     </property>
     <widget class="QWidget" name="pageWallpaper"/>
     <widget class="QWidget" name="pagePlaceholder"/>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
